src/index.js

```diff
@@ -1,7 +1,45 @@
 import './styles/style.css';
+import $ from 'jquery';
 import wrapContainer from './modules/WrapContainer.js';
 import wrapscores from './modules/wrapScores.js';
 
 wrapContainer();
 
-wrapscores();
\ No newline at end of file
+// ZEIiDb0Iy2wNzVD068jm
+
+const inputName = document.getElementById('name');
+const inputScore = document.getElementById('scoreIn');
+const submitBtn = document.getElementById('Submit');
+const refreshBtn = document.getElementById('refresh');
+
+submitBtn.addEventListener('click', () => {
+  const score = {
+    user: inputName.value,
+    score: inputScore.value,
```

Comment:
- [ ] [optional] I would suggest clearing form fields after submitting the scores, It will improve the user experience of your application.  Learn more [Here](https://bobbyhadz.com/blog/javascript-clear-input-field-after-submit).